action.yml

```diff
@@ -35,8 +48,29 @@ runs:
   steps:
     - name: Commit Range
       id: commit-range
-      uses: akaihola/darker/.github/actions/commit-range@1.7.1
+      uses: ./.github/actions/commit-range
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: '3.x'
+    - name: Get pip cache dir
```

Comment:
Do we want to use the built in caching included in setup python instead of this custom pip cache solution? Seems to be the way other repos are trending and seems to work better than the cache